its/ruling/src/test/java/org/sonar/php/it/RulingHelper.java

```diff
@@ -50,16 +56,15 @@ static OrchestratorExtension getOrchestrator(Edition sonarEdition) {
     return builder.build();
   }
 
-  static Orchestrator getOrchestrator() {
+  static OrchestratorExtension getOrchestrator() {
     return getOrchestrator(Edition.COMMUNITY);
   }
 
-  static SonarScanner prepareScanner(File path, String projectKey, String expectedIssueLocation, File litsDifferencesFile) throws IOException {
+  static SonarScanner prepareScanner(File path, String projectKey, String expectedIssueLocation, File litsDifferencesFile) {
     return SonarScanner.create(path)
       .setProjectKey(projectKey)
       .setProjectName(projectKey)
       .setProjectVersion("1")
-      .setLanguage("php")
```

Comment:
Why do we remove the language?

Reply:
I also update orchestrator dependency in this PR, and this field is removed on the Scanner